.travis.yml

```diff
@@ -10,6 +10,10 @@ matrix:
       language: c
       compiler: gcc
       script: make arm_neon=1 aarch64=1
+      arch: arm64
```

Comment:
I think this line's syntax is wrong. You need `-`.

```
    - arch: arm64
```

Reply:
This is the result of the CI: https://travis-ci.org/github/lh3/minimap2/builds/685858196 . There are total 6 jobs.
But it has to be total 7 jobs.

Reply:
yeah, sorry, IDE issues :)

Reply:
with that syntax mistake, the file is parsed in such a way that arm64 config takes only one (the last) variable values. you can check the parsed values with validator: https://config.travis-ci.com/explore
when that - is put in place, it parses well and there should be 7 tests.

Reply:
I did not know the config validation page. Thanks for sharing.
Yeah, now total 7 jobs here. https://travis-ci.org/github/lh3/minimap2/builds/685864104

Reply:
If we use `name:` syntax in `.travis.yml`, technically we can distinguish the similar jobs by the job names on the CI page. But I assume Heng likes the minimum config, and does not like the name syntax. So, maybe the current config is okay.